Assets/Scripts/State/GridTile.cs

```diff
@@ -12,14 +12,17 @@ public enum WallType
     [System.Serializable]
     public class GridTile
     {
-        public WallType Type { get; }
+        public WallType Type;
+        public WallBehaviour Wall;
 
+        public bool IsEmpty => Type == WallType.Empty;
         public bool IsDestructible => Type == WallType.WallDestructible;
         public bool IsIndestructible => Type == WallType.WallIndestructible;
 
         public GridTile(WallType type)
         {
             Type = type;
+            Wall = null;
```

Comment:
presne tomuto som sa chcel vyhnut  ze je tile kde je stena ale ma null pre ref na objekt, 

mal by si teraz oznacit Wall za nullable a vsade v kode to osetrovat 
 ale je to ohubu a ani to nerobis pri hitnuti wall

Reply:
opat, co tam chces mat, nic?